src/components/form/new.vue

```diff
@@ -248,13 +248,13 @@ export default {
         });
     },
     removeLearnMore(value) {
-      return value.replace(/Learn more: (https?:\/\/xlsform\.org\/[^.]*)\.?/, '').trim();
+      return value.replace(/Learn more: (https?:\/\/xlsform\.org[^.]*)\.?/, '').trim();
```

Comment:
I think it'd be nice to verify the character after .org. (Wouldn't want to match on xlsform.org.uk. 🤣) How about `\.org[/#]`?

Reply:
then it won't match with just `https://xlsform.org`